### PR TITLE
In this commit: Reconfigured API to handle Postgres

### PR DESCRIPTION
**Pull request by jpizzo**
Some changes to the schema, and table queries to handle the PostgreSQL syntax.

### CODE REVIEW

File: database/createReviews.js [hunk 1]
```diff
@@ -6,13 +6,26 @@ const writeReviews = fs.createWriteStream('seed/reviewComments.csv', {
...
 // populate 10K different review comments
 const reviews = { dates: [], comments: [] };
 for (let i = 0; i < 10000; i += 1) {
   reviews.dates.push(JSON.stringify(faker.date.between('2015-01-01', '2018-06-02')).slice(1, 11));
   reviews.comments.push(faker.fake('{{lorem.sentences}}'));
 }
+// populate 100K different users
+const users = {
+  id: [],
+  first_name: [],
```
Comment (lhbrennan) [inline database/createReviews.js:19]: small nitpick -- if it's an array of first_names, maybe it's better to use the variable name `first_names`? Also, why not user camelCase here?

File: database/createReviews.js [hunk 2]
```diff
@@ -6,13 +6,26 @@ const writeReviews = fs.createWriteStream('seed/reviewComments.csv', {
...
+  last_name: [],
+  user_name: [],
+};
+for (let i = 1; i < 100001; i += 1) {
+  users.id.push(i);
```
Comment (lhbrennan) [inline database/createReviews.js:24]: another super small nitpick -- might be cleaner to run the loop from 0 to 100000, but just do `users.id.push(i+1)`

File: database/index.js [hunk 3]
```diff
@@ -1,13 +1,36 @@
...
+// const pool = new Pool(conObject);
+const pgConnection = new Client({
+  user: 'jonathanpizzolato',
+  host: 'localhost',
+  database: 'dev_airbnb_sdc',
+  password: 'devadmin',
+  port: 5432,
+});
+
+pgConnection.connect((err) => {
```
Comment (lhbrennan) [inline database/index.js:14]: good error handling. i need to implement the same.

File: server/server.js [hunk 4]
```diff
@@ -19,94 +19,66 @@ app.use('/rooms/:id', express.static(path.join(__dirname, '../client/dist')));
 // Decided not to use a router, considering there are only two routes.
```
Comment (lhbrennan) [inline server/server.js:19]: I really love these notes that you leave for yourself and the reviewer (the one re: cors, the one about the router, etc.). this is a great habit. i'm going to steal it.

File: server/server.js [hunk 5]
```diff
@@ -19,94 +19,66 @@ app.use('/rooms/:id', express.static(path.join(__dirname, '../client/dist')));
...
+app.post('/api/listings/:id/reviews/new', (req, res) => {
+  const review = req.data; // Listing data (IDs) will have to be part of the data
+  query.listingReviews.post(review)
+    .then(({ rows }) => {
+      const formattedReviews = dataHandlers.processReviewsArray(rows);
+      res.status(200).send(JSON.stringify(formattedReviews));
+    })
+    .catch((error) => {
+      res.status(500).send(JSON.stringify(error));
+    });
```
Comment (lhbrennan) [inline server/server.js:55]: the routes are all very clean. one thing that confused me initially is that you chose to require your queries.js file as 'query'. i would have gone with 'queries'.

File: server/server.js (hunk 4 above)
Comment (jpizzo) [reply server/server.js:19]: haha...you need to thank Alex. Those are his original notes. But I agree they are super helpful.

**Pull request merged**